x-pack/legacy/plugins/apm/public/utils/formatters.ts

```diff
@@ -245,3 +249,145 @@ export const getFixedByteFormatter = memoize((max: number) => {
 
   return bailIfNumberInvalid(formatter);
 });
+
+/**
+ * Returns the timezone set on momentTime.
+ * (UTC+offset) when offset if bigger than 0.
+ * (UTC-offset) when offset if lower than 0.
+ * @param momentTime Moment
+ */
+function formatTimezone(momentTime: moment.Moment) {
+  const DEFAULT_TIMEZONE_FORMAT = 'Z';
+
+  // Adds plus sign when offsetHours is greater than 0.
+  const getCustomTimezoneFormat = (offsetHours: number) =>
+    offsetHours > 0 ? `+${offsetHours}` : offsetHours;
+
+  const utcOffsetHours = momentTime.utcOffset() / 60;
+
+  const utcOffsetFormatted = Number.isInteger(utcOffsetHours)
+    ? getCustomTimezoneFormat(utcOffsetHours)
+    : DEFAULT_TIMEZONE_FORMAT;
+
+  return momentTime.format(`(UTC${utcOffsetFormatted})`);
```

Comment:
off-topic: I'm starting to wonder whether it even makes sense to have a custom timezone format. I did it because I thought the standard was too long (UTC+02:00 vs UTC+2) but not sure it's a good idea to fiddle with these things. Either way, let's make a mental note and remove it if it causes confusion/problems for end users.

Reply:
I guess we are not the only ones to use the shorthand:
![image](https://user-images.githubusercontent.com/209966/68024825-83104700-fcab-11e9-87f3-deabd7e2bbdb.png)